index.js

```diff
@@ -12,3 +12,69 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   navMenu.classList.remove('active');
   body.classList.remove('active');
 }));
+
+// Dynamic Featured speakers
+
```

Comment:
- [ ] In your about page for the desktop version I think you just have the last footer with different background so kindly delete the partners section from the about page in the desktop version and keep the last footer with changing the background color to dark🔥🙏🏻✨✅

- Your design
![image](https://user-images.githubusercontent.com/58553711/210465071-53e4d751-0650-4252-b50c-c1566a1d6173.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/210465125-ac4fcd99-708b-4bf6-a708-9a68151a5ebc.png)